website/src/documentation/runners/spark-ss.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: section
+title: "Apache Spark Structured Streaming Runner"
+section_menu: section-menu/runners.html
+permalink: /documentation/runners/spark-ss/
+redirect_from: /learn/runners/spark-ss/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Using the Apache Spark Structured Streaming Runner
+
+The Apache Spark Structured Streaming Runner can be used to execute Beam pipelines using [Apache Spark Structured Streaming](https://spark.apache.org/docs/latest/structured-streaming-programming-guide.html) framework.
+The Spark Runner can execute Spark pipelines just like a native Spark application; deploying a self-contained application for local mode, running on Spark's Standalone RM, or using YARN or Mesos.
+It is still experimental, its coverage of the Beam model is partial.
```

Comment:
Please, do this more visible on the page (like a separate paragraph created with "\*\*Note:\*\*" tag).

Reply:
+1